cypress/integration/unit/logic/schedule-parser/schedule.parser.spec.ts

```diff
@@ -38,13 +38,13 @@ const babysitterSection = [
 ];
 
 const exampleData = [
-  ["Dni miesiąca", ...dates.map((x) => x.toString())],
-  emptyRow,
-  ["Dzieci", ...dates.map((x) => "1")],
-  emptyRow,
-  ...nurseSection,
-  emptyRow,
-  ...babysitterSection,
+  [
+    ["Grafik ", "", "", ""],
```

Comment:
ustalmy czy mamy tą linijkę w pliku,
tu testujemy jakby jej nie było a w przykładach nadal jest

dajcie mi znać co ustalicie = > to kwestia kilku linijek zmian w parserze

Reply:
Linijka nie powinna być wymagana przy parsowaniu, jeśli będzie to duży problem, to można narzucić jakąś mało problematyczną jej formę.